charts/common/README.md

```diff
@@ -93,6 +93,7 @@ Highlighted features:
 | postgres.credentialsSecret | string | `nil` | Override name for credentials secret. This must at least contain `PGUSER` and `PGPASSWORD`. |
 | postgres.enabled | bool | false | Enable or disable the proxy |
 | postgres.memory | int | 16 | Configure memory request for proxy without units, `Mi` inferred |
+| postgres.memoryLimit | int | 16 | Configure memory request for proxy without units, `Mi` inferred |
```

Comment:
Out of sync with actual doc. Please run

`docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest` from root of repo